routers/posts.py

```
@@ -0,0 +1,29 @@
+    return {"message": "게시글 작성"}
+
+@router.get("/me")
+def get_my_posts():
+    return {"message": "내가 쓴 게시글 목록"}
+
+@router.get("/{post_id}")
+def get_post(post_id: str):
+    return {"message": f"{post_id}번 상세 조회 (조회수 증가 로직 예정)"}
+
+@router.patch("/{post_id}")
+def update_post(post_id: str, data: PostUpdate):
```

Ruff (0.14.11)

Unused function argument: `data`

(ARG001)